core/common/src/thrift/file_system_worker.thrift

```diff
@@ -12,9 +12,13 @@ struct CloseUfsFileTOptions {
 struct CompleteUfsFileTOptions {
   1: optional string user
```

Comment:
we should probably replace user with owner everywhere, but it might make sense to do that in a separate PR (for instance, we can't simply rename this field because it would break backwards compatibility ... so we will need to add a new one a support both until support for `user` is removed)